### PR TITLE
CX-5282 Convert Uploader styles to Sass

### PR DESCRIPTION
**Pull request by kopijunkie**
Convert Uploader component styles to Sass

### CODE REVIEW

File: src/components/Uploader/style.scss [hunk 1]
```diff
@@ -22,19 +22,18 @@
...
   @media (--small-viewport) {
-    font-size: @font-size-base;
-    margin: 14*@unit 24*@unit;
+    font-size: $font-size-base;
+    margin: 14*$unit 24*$unit;
   }
 }
 
 .error {
-  &:extend(.text);
```
Comment (josokinas) [inline src/components/Uploader/style.scss:36]: You no longer need this?

File: src/components/Uploader/style.scss [hunk 2]
```diff
@@ -144,7 +143,7 @@
 }
 
 .proofOfAddressIcon {
-  &:extend(.identityIcon);
+  @extend .identityIcon;
```
Comment (josokinas) [inline src/components/Uploader/style.scss:146]: I don't think you're allowed to do that, linting does not complain? You should instead use classnames to add both classes on JSX.

File: src/components/Uploader/style.scss [hunk 3]
```diff
@@ -121,21 +120,21 @@
 }
 
 .faceIcon {
-  &:extend(.icon);
+  @extend .icon;
```
Comment (josokinas) [inline src/components/Uploader/style.scss:123]: And same here - IMO you should add both classes on JSX.

File: src/components/Uploader/style.scss (hunk 2 above)
Comment (kopijunkie) [reply src/components/Uploader/style.scss]: Have changed that in my latest commit. Didn't spot it initially as it was changed before your stylelint setup got merged in.

File: src/components/Uploader/style.scss (hunk 1 above)
Comment (kopijunkie) [reply src/components/Uploader/style.scss:36]: I checked against the existing one & how the UI looks in Surge Latest, that class doesn't exist in the stylesheet & the screen itself doesn't have anything with that class generated for the errors. Might have been leftover code.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Uploader/style.scss (hunk 3 above)
Comment (kopijunkie) [reply src/components/Uploader/style.scss]: Also fixed in latest commit(s) now. :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1078-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1078-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**